src/golang/cmd/server/main.go

```diff
@@ -17,8 +19,10 @@ var (
 		"",
 		"The path to .yml config file",
 	)
-	expose = flag.Bool("expose", false, "Whether you want to expose the server to the public.")
-	port   = flag.Int("port", connection.ServerInternalPort, "The port that the server listens to.")
+	expose        = flag.Bool("expose", false, "Whether the server will be exposed to the public.")
+	verbose       = flag.Bool("verbose", false, "Whether all logs will be shown in the terminal.")
+	port          = flag.Int("port", connection.ServerInternalPort, "The port that the server listens to.")
+	serverLogPath = filepath.Join(os.Getenv("HOME"), ".aqueduct", "server", "logs", "server_log")
```

Comment:
Nit: Let's make this page `~/.aqueduct/logs/server`. We might eventually want to unify our logging here (e.g., add operator logs separately as well).

Reply:
@vsreekanti the operator log files are already in `~/.aqueduct/server/logs/<OPERATOR_NAME>`. That's why I put the server log in this same directory as well. Do you still wanna change this to the new location? If so for consistency we also want to move the operator logs to the new directory, which may be tricky because existing workflow's logs were in the old directory and we may need to move them.

Reply:
Got it, let's rename the file to `server` and then leave this directory.